lib/pronto/formatter/bitbucket_pull_request_formatter.rb

```diff
@@ -1,6 +1,27 @@
 module Pronto
   module Formatter
     class BitbucketPullRequestFormatter < PullRequestFormatter
+      def format(messages, repo, patches)
+        client = client_module.new(repo)
+        existing = existing_comments(messages, client, repo)
+        comments = new_comments(messages, patches)
+        additions = remove_duplicate_comments(existing, comments)
+
+        remove_comments(client, existing, comments) if existing.any?
```

Comment:
This should be optional. I'm not sure how we'd go about it, maybe a separate `PullRequestFormatter` which deletes comments?